### DISCUSSION TITLE
Next release for Collector layer?

### DISCUSSION THREAD

**Discussion by juanitosvq** [open-telemetry/opentelemetry-lambda#818] [General]
Hi there!

We've been using the ADOT layers and we just found out about this project. We are very excited to use it, we were wondering if there are any plans to release a new version with the latest otel collector (I saw the code is already merged: https://github.com/open-telemetry/opentelemetry-lambda/pull/796). 

Thanks,
Juan

**Comment by tylerbenson** [comment]
Thanks for the reminders.  I published the release just now:
https://github.com/open-telemetry/opentelemetry-lambda/releases/tag/layer-collector%2F0.2.0
